# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../man-pages/inst-min.patch
# Copyright (C) 2009 The OpenSDE Project
# Copyright (C) 2004 - 2006 The T2 SDE Project
# Copyright (C) 1998 - 2003 Clifford Wolf
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---


Only install the new stuff ...

 - Clifford

--- ./Makefile.orig	2009-07-25 09:53:21.000000000 +0300
+++ ./Makefile	2009-08-13 21:40:45.000000000 +0300
@@ -19,10 +19,8 @@
 screen:
 	-mkdir not_installed
 	for i in man?/*; do \
-		if [ $(MANDIR)/"$$i" -nt "$$i" ]; then \
-			cmp -s $(MANDIR)/"$$i" "$$i" > /dev/null 2>&1; \
-			if [ "$$?" != 0 ]; then mv "$$i" not_installed; fi; \
-		fi; \
+ 		[ -f "$(MANDIR)/$$i" ] && mv -v $$i not_installed; \
+ 		true; \
 	done
 
 remove:
@@ -53,6 +51,7 @@
 	for i in man?; do \
 		install -d -m 755 $(DESTDIR)$(MANDIR)/"$$i" || exit $$?; \
 		install -m 644 "$$i"/* $(DESTDIR)$(MANDIR)/"$$i" || exit $$?; \
+		true; \
 	done; \
 
 # someone might also want to look at /var/catman/cat2 or so ...
